artifacts-guidance.md

```diff
@@ -1,6 +1,30 @@
 # Guidance for Artifacts Authors
 
-Content other than OCI container images MAY be packaged using the image manifest.
-When this is done, the `config.mediaType` value should not be a known OCI image config [media type](media-types.md).
-Historically, due to registry limitations, some tools have created non-OCI conformant artifacts using the `application/vnd.oci.image.config.v1+json` value for `config.mediaType` and values specific to the artifact in `layer[*].mediaType`.
+## Artifacts and Images
+
+This specification is primarily concerned with packaging two kinds of content: Artifacts and Images.
+Both are representing using a [manifest](manifest.md).
+Images are defined in this specification as conformant content with a conformant [config](config.md) processed according to [conversion.md](conversion.md) to derive a [runtime-spec][] configuration blob.
+Conversely, an Artifact is any other conformant content that **does not contain a config to be interpreted by a runtime-spec implementation using the conversion mechanism.
+
+## Creating an Artifact
+
+Content other than Images MAY be packaged using the [manifest]; this is otherwise known as an Artifact.
+When this is done, the `artifactType` should be set to a custom media type, or the `config.mediaType` should not be a known Image config [media type](media-types.md).
 Implementation details and examples are provided in the [image manifest specification](manifest.md#guidelines-for-artifact-usage).
+
+Note: Historically, due to registry limitations, some tools have created non-conformant Artifacts using the `application/vnd.oci.image.config.v1+json` value for `config.mediaType`.
+
+## Interacting with Artifacts
+
+Runtimes implementing the [runtime-spec][] and following the process described in [conversion.md](conversion.md) SHOULD ignore unknown Artifacts (as determined by the presence of a descriptor `artifactType`) when selecting content from an [index](image-index.md).
+It is possible that software implementing the runtime-spec may also be able to interpret Artifacts; however that is outside the scope of this spec.
```

Comment:
I'd say implementation-defined; I think that if we tried to standardize a way to interpret certain artifacts in the OCI spec itself it would be in conversion.md and the runtime-spec.

Reply:
This makes sense. I could see some type of guidance on what must/should get converted to the runtime spec.